docs/build/manifest.md

```diff
@@ -514,6 +514,13 @@ filter:
   specVersion: [null, 23] # Index block with specVersion less than or equal 23.
 ```
 
+The `modulo` filter allows handling every N blocks, this is useful if you want to group or calculate data at a set interval. The folloing example shows how to use this filter.
+
+```yml
+filter:
+  modulo: 50 # Index every 50 blocks: 0, 50, 100, 150....
+```
+
```

Comment:
@stwiname Good job. I would change it a little bit, link sentences together.  Also there is a minor typo. 

The `modulo` filter allows handling every N blocks, which is useful if you want to group or calculate data at a set interval. The following example shows how to use this filter.